cfn-resources/test/e2e/utility/atlas_helper.go

```diff
@@ -57,14 +59,85 @@ func NewMongoDBClient() (atlasClient *admin.APIClient, err error) {
 		c.BaseURL = baseURL
 	}
 	// New SDK Client
-	sdkV2Client, err := c.NewSDKV2Client(client)
+	sdkV2Client, err := c.NewSDKV2LatestClient(client)
 	if err != nil {
 		return nil, fmt.Errorf("unable to create Atlas client")
 	}
 
 	return sdkV2Client, nil
 }
 
+func CreateProject(t *testing.T, atlasClient *admin.APIClient, orgID, projectName string) string {
```

Comment:
it is just curious that we didn't have this code implemented in CFN to create resources for tests. Well done @lantoli @AgustinBettati!! - Am I correct to say this is the first e2e test we're writing for CFN?

Reply:
this is the second one, the first one for project resource was created by @maastha 

the idea we're following is to test in CFN the specific resource, and get the prerequisites (like project, cluster) ready  for it to use

Reply:
Awesome @maastha!